style.css

```diff
@@ -0,0 +1,104 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins:200i,400&display=swap');
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  font-family: 'Poppins', sans-serif;
+  font-size: 1rem;
+  color: #f3f3f3;
+  margin: 3.1rem auto;
+  width: 100%;
+  max-width: 720px;
+  background-image: linear-gradient(64deg, rgb(87 87 98 / 80%), rgba(136, 136, 206, 0.7)), url(https://images.unsplash.com/photo-1547036967-23d11aacaee0?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1904&q=80);
+  background-repeat: no-repeat;
+  background-size: cover;
+  line-height: 1.4;
+  font-weight: 400;
+}
+
+header {
+  margin-bottom: 2rem;
+}
+
+h1,
+p {
+  margin-bottom: 0.5rem;
+}
+
+#title {
+  font-weight: 500;
+}
+
+#description {
+  font-style: italic;
+  font-weight: 200;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  border-radius: 0.5rem;
+  padding: 2rem;
+  background-color: #1b1b32cc;
+}
+
+.form-group {
+  display: flex;
+  flex-direction: column;
+  text-align: left;
+  margin: 0 0 1.25rem 0;
+  padding: 0.25rem;
+}
+
+span {
+  margin-left: 0.25rem;
+  font-size: 0.9rem;
+}
+
+label {
+  display: flex;
+  align-items: center;
+  font-size: 1.12rem;
+  margin-bottom: 0.5rem;
+}
+
+input,
+select,
+button,
+textarea {
+  padding: 0.5rem;
+  border-radius: 0.4rem;
+  font-size: inherit;
+  line-height: inherit;
+}
+
+.radio-btn,
+.checkbox {
+  display: inline-block;
+  margin-right: 0.625rem;
+  min-height: 1.25rem;
+  min-width: 1.25rem;
+}
+
+.input-textarea {
+  min-height: 120px;
+  width: 100%;
+  padding: 0.625rem;
+}
+
+#submit {
```

Comment:
I think it would be better to add :hover functionality to submit button.

Reply:
I have made changes as per requirement. [you can check here](https://github.com/MNisarAli/Proj-Survey-form/pull/2).